internal/cypress/sauce/runner.go

```diff
@@ -78,31 +86,48 @@ func (r *Runner) runSuites(fileID string) int {
 
 	// Collect results.
 	errCount := 0
+	completed := 0
+	total := len(r.Project.Suites)
+	inprogress := total
+
+	log.Info().Msgf("Suites completed: %d in progress: %d", completed, inprogress)
 	for i := 0; i < len(r.Project.Suites); i++ {
-		r := <-results
-		if r.err != nil {
-			log.Err(r.err).Str("suite", r.suiteName).Msg("Suite failed.")
+		res := <-results
```

Comment:
does results has the job status? we should be counting error vs passed to show in the summary

Reply:
yeap, it returns status.